student.rb

```diff
@@ -1,12 +1,13 @@
-require './person'
+require_relative 'person'
 
 # This is student class, which represents student's information
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :classroom
```

Comment:
- [ ] [REQUIRED] Good job so far! The `:classroom` should contained at `attr_reader` 🙏🏼 

```ruby
attr_reader :classroom
```